### PR TITLE
Initialize Project

### PR DESCRIPTION
**Pull request by RayhanTabase**
In this feature:
- Initialised React app
- Added React Redux (npm install react-redux)
- Structured application files using a "feature folder" approach
- Use of reusable components
- Routing to books and categories page

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+
+import './App.css';
```
Comment (dasileker) [inline src/App.js]: - [x] Please fix the import for the CSS it's an incorrect call for the App.css there is no such file, please take a good look to what you should call.

File: src/components/pages/books/BookItem.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import BooksDetails from './BookDetails';
+
+function BookItem() {
+  return (
+    <li className="booksItem">
+      <BooksDetails />
+    </li>
+  );
```
Comment (dasileker) [inline src/components/pages/books/BookItem.js]: - [x] This is a good start but instead of calling back the component, please use the Router.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi  @RayhanTabase,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._


- Please correct the issue regarding the import in your App.js.

opopop


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (RayhanTabase) [reply src/App.js]: I missed this. Thank you for the feedback

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @RayhanTabase, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**